src/genetic_functions/select_parents.jl

```diff
@@ -0,0 +1,28 @@
+function select_fitness_proportional_parents(population::Array{RuleNode}, fitness_array::Array{<:Real})::Tuple{RuleNode,RuleNode}
+    sum_of_fitness = sum(fitness_array)
+    fitness_array_normalized = [fitness_value / sum_of_fitness for fitness_value in fitness_array]
+    parent1 = select_chromosome(population, fitness_array_normalized)
+    parent2 = select_chromosome(population, fitness_array_normalized)
+    return parent1, parent2
+end
+
+function select_two_random_parents(population::Array{RuleNode}, fitness_array::Array{<:Real})::Tuple{RuleNode,RuleNode}
```

Comment:
What is the difference between `select_fitness_proportional_parents` and `select_two_random_parents`?

Reply:
I removed `select_two_random_parents` for now.